src/platform/packages/shared/shared-ux/chrome/navigation/src/types.ts

```diff
@@ -44,32 +44,20 @@ export interface NavigationServices {
 }
 
 /**
- * An interface containing a collection of Kibana dependencies required to
+ * An interface containing a collection of Chrome dependencies required to
  * render this component
  * @public
  */
-export interface NavigationKibanaDependencies {
```

Comment:
Technically, it is no longer convenient to accept "kibana" dependencies because we are now using a component from core (chrome). So, I had to change this.